jordan_bundy/app/sass/application.scss

```diff
@@ -0,0 +1,2 @@
+@import "base";
+
```

Comment:
You want to import ALL your partial Sass files, not just base, and all those partial files need to have an `_` as the beginning of their filename: `_base.scss`, `_layout.scss` etc.  That `_` does not get typed into your `@import` statements, however.  It's just a way for gulp-sass to know what should be a partial, and what to use as for an 'entrypoint'.  In your case, the entrypoint is application.scss (no underscore!).

Then when you run gulp-sass, it will know to catch ALL .scss partials, import them into application.scss, and then compile the whole mess into CSS, which gets handed off to the minify task.

If this doesn't make sense, come see me!